app/controllers/email_alert_subscriptions_controller.rb

```diff
@@ -31,6 +40,10 @@ def subscriber_list_params
     SubscriberListParamsPresenter.new(content, filter_params).subscriber_list_params
   end
 
+  def validate_choices
```

Comment:
I think this should be `validate_choices!`, as the convention of using `!` for exception-throwing validation stuff seems pretty common